Backpack-SwiftUI/AppSearchModal/Classes/AppSearchModalContentView.swift

```diff
@@ -111,6 +119,7 @@ struct AppSearchModalContentView: View {
     struct ItemCellButtonStyle: ButtonStyle {
         func makeBody(configuration: Configuration) -> some View {
             configuration.label
+                .background(configuration.isPressed ? .canvasContrastColor : .canvasContrastColor.withAlphaComponent(0))
```

Comment:
set the alpha to 0, to avoid a 'flicker' when using BPKColor.clear.
If we use BPKColor.Clear a darker color flicker when tapping ends.

Reply:
(Also considering if this needs to be SurfaceContrast as the Background uses surface and not canvas)